build.assets/Dockerfile

```diff
@@ -6,7 +6,44 @@
 # runtime requirement for the host to have newer glibc too. For example,
 # teleport built on any newer Ubuntu version will not run on Centos 7 because
 # of this.
-FROM ubuntu:18.04
+
+# Build libfido2 separately for isolation, speed and flexibility.
+FROM buildpack-deps:18.04 AS libfido2
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends cmake && \
+    rm -rf /var/lib/apt/lists/*
+
+# Install libudev-zero.
+# libudev-zero replaces systemd's libudev
+RUN git clone --depth=1 https://github.com/illiliti/libudev-zero.git -b 1.0.1 && \
+    cd libudev-zero && \
+    make install-static
+
+# Install libcbor.
+RUN git clone --depth=1 https://github.com/PJK/libcbor.git -b v0.9.0 && \
+    cd libcbor && \
+    cmake \
+        -DCBOR_CUSTOM_ALLOC=ON \
```

Comment:
Have we figured out what this is for?

Reply:
It seems to be their preferred setting, for example see the README: https://github.com/PJK/libcbor#compile-from-source. There are [mentions in issues](https://github.com/PJK/libcbor/issues/147#issuecomment-666401486) about it being unfortunate that it is disabled by default. I don't know much more than that, though.